src/data/ACTIONS/SMN.js

```diff
@@ -111,7 +117,99 @@ export default {
 		id: 7429,
 		name: 'Enkindle Bahamut',
 		icon: 'https://xivapi.com/i/002000/002693.png',
-		cooldown: 13,
+		cooldown: 10,
+	},
+
+	FIREBIRD_TRANCE: {
+		id: 16549,
+		name: 'Firebird Trance',
+		icon: 'https://xivapi.com/i/002000/002734.png',
+		cooldown: 60,
+	},
+
+	FOUNTAIN_OF_FIRE: {
+		id: 16514,
+		name: 'Fountain Of Fire',
+		icon: 'https://xivapi.com/i/002000/002735.png',
+		onGcd: true,
+	},
+
+	BRAND_OF_PURGATORY: {
+		id: 16515,
+		name: 'Brand Of Purgatory',
+		icon: 'https://xivapi.com/i/002000/002736.png',
+		onGcd: true,
+	},
+
+	ENKINDLE_PHOENIX: {
+		id: 16516,
+		name: 'Enkindle Phoenix',
+		icon: 'https://xivapi.com/i/002000/002737.png',
+		cooldown: 10,
+	},
+
+	//Egi Assault, Egi Assault II, and Enkindle have unique ids depending on the summoned pet.
+	EGI_ASSAULT_GARUDA: {
```

Comment:
I'd rather the keys match the name where possible, i.e. `ASSAULT_I_AERIAL_SLASH` or `ASSULT_AERIAL_SLASH`, etc.